dynamo/preprocessing/utils.py

```diff
@@ -1127,9 +1134,9 @@ def calc_new_to_total_ratio(adata: anndata.AnnData) -> Union[Tuple[np.ndarray, n
 
 
 def scale(
-    adata: anndata.AnnData,
+    adata: AnnData,
     layers: Union[List[str], str, None] = None,
-    scale_to_layer: Union[str, None] = None,
+    scale_to_layer: Optional[str] = None,
     scale_to: float = 1e6,
```

Comment:
let us change this default to 1e4 as others used

Reply:
Done